adding-autodeployment/README.md

```diff
@@ -0,0 +1,100 @@
+# Autodeployment Guide
+## What to do?
+1. [Get deployer credentials](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#1-getting-deployer-credentials)
+2. [Update `.travis.yml`](https://github.com/18F/development-guide/tree/add-autopilot/adding-autodeployment#2-update-travisyml)
```

Comment:
In order to make this broader, should this step be "Configure the continuous integration service" with "Update `.travis.yml`" as a sub-item? There may be other approaches for things like Circle CI, Concourse, or other things in use around the org.